### ISSUE TITLE
ignore function broken on add and remove users.

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#484]
**Reported by Ageless on 26 Jun 37894515 23:33 UTC**
Both adding to filter and removing from filter is broken. 
David did add !'s to the add_to_ignore code, but in [it's still broken. 
When I tested this on Cosmology@Home by adding users to ignore, or with users already on ignore changing my preferences and saving them, the screen would reload and at the top of it it says "Missing user xx", where xx is the user's ID number. 

Removing from filter in edit_forum_preferences_action.php is now:
```
$ignored_users = get_ignored_list($user);
for ($i=0;$i<sizeof($ignored_users);$i++){
    if ($_POST["remove".trim($ignored_users[$i](14289]))]!=""){
        $other_user = BoincUser::lookup_id($user_to_add);
        if (!$other_user) {
            echo "No such user: $other_user";
        } else {
            remove_ignored_user($user, $other_user);
        }
    }
}
```
With help of TJM of Enigma, this should be
```
$ignored_users = get_ignored_list($user); 
for ($i=0;$i<sizeof($ignored_users);$i++){ 
    if ($_POST[
        $other_user = BoincUser::lookup_id($ignored_users[$i]("remove".trim($ignored_users[$i])]!=""){)); 
        if (!$other_user) { 
            echo "No such user: $other_user"; 
        } else { 
            remove_ignored_user($user, $other_user); 
        } 
    } 
}
```

**Comment by romw** [comment]
**Commented by Ageless on 6 Mar 37894518 18:40 UTC**
fixing title.

**Comment by romw** [comment]
**Commented by davea on 8 Sep 37894640 03:33 UTC**
(In [14290])  - user web: fix bug in user filter; fixes #505
